pylint/exceptions.py

```diff
@@ -0,0 +1,14 @@
+# Licensed under the GPL: https://www.gnu.org/licenses/old-licenses/gpl-2.0.html
```

Comment:
Shouldn't we add a Copyright notice here? We just had some issues with Debian packaging due to the fact that we were missing Copyright notices on files. I am not sure what Copyright this file should have, yours or Logilab's? I think yours, no?